shared-plugins/vip-go-elasticsearch/vip-go-elasticsearch.php

```diff
@@ -126,7 +126,7 @@ public function filter__the_posts( $posts, $query ) {
 		if ( ! $query->is_main_query() || ! $query->is_search() )
 			return $posts;
 
-		if ( ! is_array( $this->search_result ) )
+		if ( is_wp_error( $this->search_result ) || ! is_array( $this->search_result ) || empty( $this->search_result['results'] ) || empty( $this->search_result['results']['hits'] ) )
```

Comment:
Right, which is covered by the addition of the is_wp_error check. My concern was the added check for `hits` since that's going beyond the scope of the issue. Might be better to more explicitly look for the `error` key instead? Not a huge deal since the behaviour ends up the same; just trying to prevent potential future bugs.